src/constants.js

```diff
@@ -0,0 +1 @@
+export const CHANNEL = '__glamorous__'
```

Comment:
Yeah, let's go with the third option.... I expect we'll put more in here later, but I don't want this rule slowing us down now.